packages/tasks/src/tasks/audio-to-audio/spec/input.json

```diff
@@ -0,0 +1,13 @@
+{
+	"$id": "/inference/schemas/audio_to_audio/input.json",
+	"$schema": "http://json-schema.org/draft-06/schema#",
+	"description": "Inputs for Audio to Audio inference",
+	"title": "AudioToAudioInput",
+	"type": "object",
+	"properties": {
+		"inputs": {
+			"description": "The input audio data"
+		}
```

Comment:
Let's also spec the parameters as an empty object to allow users to specify arbitrary kwargs

See this fo example:
https://github.com/huggingface/huggingface.js/blob/af447753043472d8ba65ab0591be7487433f7261/packages/tasks/src/tasks/table-question-answering/spec/input.json#L30-L43

Reply:
Will do :+1:

Reply:
done in https://github.com/huggingface/huggingface.js/pull/502/commits/bd7a46bbd06fa1a341e2b1ca4190bc5db6a32137